statsfs/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"otelstats"
+)
+
+func parseFlags() (string, string) {
+	exporterName := flag.String("exporter", "prometheus", "Exporter to use. Choose between prometheus (default), stdout and gcp")
+	statsfsPath := flag.String("statsfspath", "otelstats/testsys/kernel/stats", "Path to statsfs filesystem for metrics (default to the teset filesystem)")
```

Comment:
Normally we just define the flags with package level vars, like this:
```suggestion
var (
	exporterName = flag.String("exporter", "prometheus", "Exporter to use. Choose between prometheus (default), stdout and gcp")
	statsfsPath = flag.String("statsfspath", "otelstats/testsys/kernel/stats", "Path to statsfs filesystem for metrics (default to the teset filesystem)")
)
```

And we can call the `flag.Parse()` directly in the main function.